crates/biome_json_parser/tests/json_test_suite/err/structure_open_object_with_key_open_string.json.snap

```diff
@@ -0,0 +1,103 @@
+---
+source: crates/biome_json_parser/tests/spec_test.rs
+expression: snapshot
+---
+## Input
+
+```json
+{
+    "key": 0,
+    "a
+}
+```
+
+
+## AST
+
+```
+JsonRoot {
+    bom_token: missing (optional),
+    value: JsonArrayValue {
```

Comment:
Why the root here is a `JsonArrayValue`?

Reply:
This is what the parser does when it encounters a bogus member in an object.
I didn't change this behavior, you can find multiple snapshots with this choice.

I share your point, we should handle this differently.
However, this could be addressed in another PR.